org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/hover/FocusableBrowserInformationControl.java

```diff
@@ -128,7 +128,7 @@ protected void createContent(Composite parent) {
 	private static Object safeEvaluate(Browser browser, String expression) {
 		try {
 			return browser.evaluate(expression);
-		} catch (Throwable ex) {
+		} catch (Exception ex) {
```

Comment:
I know some tools discourage catching Throwable, but at this point, I think it is what we want. No matter what it happens, we want to log it and continue as normal.

Reply:
I am not sure what is meant by "swallow", but it will catch the OOM and log it. Maybe it cannot log an OOM because the logging code might not be able to allocate the needed memory, but in this case probably reraising will not help much either.
In any case, I think for this case we can catch Exception as well. So I think it is fine.